python/pyspark/errors/error_classes.py

```diff
@@ -39,6 +39,11 @@
       "Function `<func_name>` should return Column, got <return_type>."
     ]
   },
+  "JVM_ATTRIBUTE_NOT_SUPPORTED" : {
+    "message" : [
+      "Attribute `<attr_name>` is not supported in Spark Connect as it depends on the JVM. If you need to use this attribute, do not use Spark Connect when creating your session."
```

Comment:
cc @gatorsmile @grundprinzip do you have any suggestion on this error message?

Reply:
LGTM